google/cloud/bigtable/internal/default_row_reader.cc

```diff
@@ -107,8 +109,13 @@ absl::variant<Status, bigtable::Row> DefaultRowReader::Advance() {
     if (!last_read_row_key_.empty()) {
       // We've returned some rows and need to make sure we don't
       // request them again.
-      row_set_ =
-          row_set_.Intersect(bigtable::RowRange::Open(last_read_row_key_, ""));
+      if (reverse_) {
+        row_set_ = row_set_.Intersect(
+            bigtable::RowRange::Open("", last_read_row_key_));
```

Comment:
Same comment as above.  For all I know this is the "Right Thing". If it is, it deserves a comment explaining why.

Reply:
Ditto